frontend/src/scenes/session-recordings/player/commenting/playerFrameCommentOverlayLogic.ts

```diff
@@ -104,15 +98,16 @@ export const playerCommentOverlayLogic = kea<playerCommentOverlayLogicType>([
             }, 250)
 
             try {
-                const apiPayload = {
-                    date_marker: dayjs(values.currentTimestamp).toISOString(),
+                await api.comments.create({
                     content: emoji,
-                    scope: AnnotationScope.Recording,
-                    recording_id: props.recordingId,
-                    is_emoji: true,
-                }
-                const createdAnnotation = await api.annotations.create(apiPayload)
-                actions.appendAnnotations([createdAnnotation])
+                    scope: 'recording',
+                    item_id: props.recordingId,
+                    item_context: {
+                        is_emoji: true,
+                        time_in_recording: dayjs(values.currentTimestamp).toISOString(),
+                    },
```

Comment:
it's nice that `Comment` has this `item_context` dict field so we can store this useful info instead of having to have a db migration 👨‍🍳 👌